homeassistant/components/tessie/config_flow.py

```diff
@@ -0,0 +1,105 @@
+"""Config Flow for Tessie integration."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from http import HTTPStatus
+from typing import Any
+
+from aiohttp import ClientConnectionError, ClientResponseError
+from tessie_api import get_state_of_all_vehicles
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ACCESS_TOKEN
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+TESSIE_SCHEMA = vol.Schema({vol.Required(CONF_ACCESS_TOKEN): str})
+
+
+class TessieConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config Tessie API connection."""
+
+    VERSION = 1
+    DOMAIN = DOMAIN
+
+    def __init__(self) -> None:
+        """Initialize."""
+        self._reauth_entry: ConfigEntry | None = None
+
+    async def async_step_user(
+        self, user_input: Mapping[str, Any] | None = None
+    ) -> FlowResult:
+        """Get configuration from the user."""
+        errors = {}
+        if user_input and CONF_ACCESS_TOKEN in user_input:
```

Comment:
I dont think the if CONF_ACCESS_TOKEN in user_input is needed, as it is required in the schema

Reply:
user_input can be null, for example when the form is first opened.

Reply:
Oh I see what your saying, let me try that.